apis/python/src/tiledbsoma/tiledb_object.py

```diff
@@ -74,24 +61,25 @@ def delete(self) -> None:
 
     def __repr__(self) -> str:
         """
-        Fallback string display. Will be overridden by any interesting subclasses.
+        Default repr
         """
-        return f"name={self._name},uri={self._uri}"
-
-    def _repr_aux(self) -> Sequence[str]:
-        raise Exception("Must be overridden by inherting classes.")
+        if self.exists():
+            return f"{self.type}(uri={self._uri})"
```

Comment:
suggested:

```diff
-            return f"{self.type}(uri={self._uri})"
+            return f'{self.type}(uri="{self._uri}")'
```

Reply:
this matches the key being quoted, & jives with the idea that a `repr` of an object should give some notion of how to construct the object

Reply:
I don't believe there is any such requirement on repr (and most reprs don't follow this suggestion as it is overly verbose).  I can provide you with numerous examples from widely used packages if you desire (tiledb, numpy, pandas, etc).

However, in the name of landing this thing, I'll push this change momentarily.